test/controllers/glossary_terms_controller_test.rb

```diff
@@ -281,17 +281,25 @@ def test_create_invalid_name_with_image
   def test_create_image_save_failure
     login
     # Simulate image.save failure.
-    Image.any_instance.stubs(:save).returns(false)
-    post(:create, params: term_with_image_params)
-
+    image = images(:disconnected_coprinus_comatus_image)
```

Comment:
I'm stubbing a random image. 
Should I do something else.
NOTE: This is getting to the right place in the case, and the assertion passes.

Reply:
This seems fine to me.